src/components/Header/Header.css

```diff
@@ -0,0 +1,12 @@
+.header {
+  height: 15vh;
```

Comment:
I think this is an excellent height for the header. Not too big, so it's not in the way, but not so small that it's unnoticeable.